float32 current_1
float32 current_2
float32 current_3
float32 current_4
float32 battery
